contrib/lib/Cargo.toml

```diff
@@ -37,6 +45,21 @@ handlebars = { version = "0.32", optional = true }
 glob = { version = "^0.2", optional = true }
 tera = { version = "0.11", optional = true }
 
+# Database dependencies
+diesel = { version = "1.0", default-features = false, optional = true }
+postgres = { version = "0.15", optional = true }
+r2d2 = { version = "0.8", optional = true }
+r2d2_postgres = { version = "0.14", optional = true }
+mysql = { version = "14", optional = true }
+r2d2_mysql = { git = "https://github.com/ELD/r2d2-mysql.git", branch = "update-r2d2-mysql", optional = true }
```

Comment:
This points to a fork for now. I updated the crate to use a more up-to-date version of `mysql` that doesn’t run into compilation troubles. I have a pending PR on the original repo.